internal/compiler/parser.rs

```diff
@@ -374,8 +374,8 @@ declare_syntax! {
         Expression-> [ ?Expression, ?FunctionCallExpression, ?IndexExpression, ?SelfAssignment,
                        ?ConditionalExpression, ?QualifiedName, ?BinaryExpression, ?Array, ?ObjectLiteral,
                        ?UnaryOpExpression, ?CodeBlock, ?StringTemplate, ?AtImageUrl, ?AtGradient, ?AtTr,
-                       ?MemberAccess ],
-        /// Concatenate the Expressions to make a string (usually expanded from a template string)
+                       ?MemberAccess, ?Predicate ],
+        /// Concatenate the Expressions to make a string (usually expended from a template string)
```

Comment:
I think the previous spelling of expanded was correct. (Maybe a merge conflict?)